sc/social/like/locales/pt_BR/LC_MESSAGES/sc.social.like.po

```diff
@@ -361,22 +369,32 @@ msgid "help_canonical_url"
 msgstr "A URL canónica do objeto que será usada como seu ID permanente no gráfico (<code>og:url</code>)."
 
 #. Default: "If enabled, the site will not provide advanced sharing widgets , instead simple links will be used.\nThis will limits user experience and features (like the share count) but will enhance users privacy: no 3rd party cookies will be sent to users."
-#: sc/social/like/interfaces.py:110
+#: sc/social/like/interfaces.py:113
 msgid "help_do_not_track"
 msgstr "Se habilitado, o site no provera os widgets avançados, mas simples links para os serviços. Isso limitará a experiência de usuário e as funcionalidades (como a conta de compartilhamentos) mas respeitará a privacidade dos usuários: nos se enviaram cookies a terceiros."
 
 #. Default: "Please select which plugins will be used"
-#: sc/social/like/interfaces.py:77
+#: sc/social/like/interfaces.py:80
 msgid "help_enabled_plugins"
 msgstr "Informe os plugins que serão usados"
 
+#. Default: "If enabled, an event is triggered to make Facebook crawler scrape and cache metadata every time a new piece content is published and every time published content is edited. This will keep the metadata updated on Facebook always."
+#: sc/social/like/interfaces.py:229
+msgid "help_facebook_prefetch_enabled"
+msgstr "Se habilitado, se ativará um evento para o crawler do Facebook atualizar e cachear a metadata cada vez que um novo conteúdo seja publicado e cada vez que um novo conteúdo publicado seja editado."
+
+#. Default: "Content without a lead image will use this image as fallback (<code>og:image</code> property). There could be a delay of up to 2 minutes when replacing this image."
+#: sc/social/like/interfaces.py:149
+msgid "help_fallback_image"
+msgstr "O conteúdo que não possua uma imagem principal usará está imagem como propriedade <code>og:image</code>. Pode ter uma demora de até 2 minutos quando a imagem for substituída."
```

Comment:
"O conteúdo que não possua uma imagem principal usará esta imagem como propriedade <code>og:image</code>. Pode ter uma demora de até 2 minutos quando a imagem for substituída."